src/aap_eda/api/serializers/activation.py

```diff
@@ -403,6 +403,7 @@ class Meta:
         required=False,
         allow_null=True,
         child=serializers.IntegerField(),
+        validators=[validators.check_multiple_credentials],
```

Comment:
To align with how you name other validator methods, this one may be `validators.check_credential_types_for_activation`

Reply:
The name has been used already: https://github.com/ansible/eda-server/pull/882/files#diff-0fc6d501fb19fb05efa10f75a58fc762f5aec1ecf8ecd80fa488d619f346b626R70